app/components/common/cityModule/cityElectionsController.js

```diff
@@ -1,11 +1,6 @@
 'use strict';
 
 function cityElectionsController($scope, ballot) {
-  $scope.ballot = ballot;
-
-  $scope.isOffice = isOffice;
-  $scope.isReferendum = isReferendum;
-
```

Comment:
This is here as per @jnmarcus 's styleguide. https://github.com/johnpapa/angular-styleguide#bindable-members-up-top

But instead you should disable the `latedef` jshint option.